### DISCUSSION TITLE
Mongolian not accepted on my computer

### DISCUSSION THREAD

**Discussion by FiableDotBiz** [espeak-ng/espeak-ng#1728] [Q&A]
Hello.

I read on Wikipedia that espeak-ng reads Mongolian, but on my computer, with version 1.51.1 (Fedora workstation 38, Gnome 44.1), I get:
```
$ espeak-ng -v mn гарах
Error: The specified espeak-ng voice does not exist.
```

What should I do? Thank you in advance.

**Comment by jaacoppi** [comment]
Monogolian is not supported. Can't find any references to it in source code.

**Comment by btseee** [comment]
Dear @FiableDotBiz,

I added Mongolian language by this PR  #2202  you should be able to use it now.
